diff --git a/dist/index.js b/dist/index.js
index f9b5cf70f817015cf60d752543c22fcf067db05c..1f3ee92a187779c37005a9ebbd128d16abedbbdb 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,4 +1,3 @@
-import { loadPlugins } from "./load-plugins.js";
 import createCollectHeadings from "./rehype-collect-headings.js";
 import rehypeEscape from "./rehype-escape.js";
 import rehypeExpressions from "./rehype-expressions.js";
@@ -19,7 +18,11 @@ import markdownToHtml from "remark-rehype";
 import { unified } from "unified";
 import { VFile } from "vfile";
 export * from "./types.js";
-const DEFAULT_REMARK_PLUGINS = ["remark-gfm", "remark-smartypants"];
+
+import remarkGfm from 'remark-gfm';
+import smartypants from 'remark-smartypants';
+
+const DEFAULT_REMARK_PLUGINS = [[remarkGfm], [smartypants]];
 const DEFAULT_REHYPE_PLUGINS = [];
 async function renderMarkdown(content, opts) {
   var _a;
@@ -41,8 +44,8 @@ async function renderMarkdown(content, opts) {
     remarkPlugins = [...DEFAULT_REMARK_PLUGINS, ...remarkPlugins];
     rehypePlugins = [...DEFAULT_REHYPE_PLUGINS, ...rehypePlugins];
   }
-  const loadedRemarkPlugins = await Promise.all(loadPlugins(remarkPlugins));
-  const loadedRehypePlugins = await Promise.all(loadPlugins(rehypePlugins));
+  const loadedRemarkPlugins = DEFAULT_REMARK_PLUGINS;
+  const loadedRehypePlugins = DEFAULT_REHYPE_PLUGINS
   loadedRemarkPlugins.forEach(([plugin, pluginOpts]) => {
     parser.use([[plugin, pluginOpts]]);
   });